SatoshiHeroFront/src/app/app.component.html

```diff
@@ -1,2 +1,11 @@
-<link rel="stylesheet" href="/SatoshiHeroClone/src/styles.css">
-<app-sidebar></app-sidebar>
\ No newline at end of file
+<link rel="stylesheet" href="/SatoshiHeroClone/src/styles.css" />
+<app-dropdown
+  [Items]="[
+    'BTC:00000000002',
+    'EF:2222222222',
+    'RUB:222222',
+    'BLR:22222222222'
+  ]"
+  [SelectedItem]="'BTC:00000000002'"
+></app-dropdown>
```

Comment:
maybe move single > to `[SelectedItem]="'BTC:00000000002'"` and maybe  `</app-dropdown>`  up to it?

Reply:
Yep, I agree. I use some default code styler in VS Code. Can you advise some code styler or config file for angular in VS Code?